### PR TITLE
DealershipDB Finished.

### PR DESCRIPTION
**Pull request by astrohlicpu**
Hi,
It was a pleasure to work on your specs. Please look over my work and let me know if you want anything changed.

### CODE REVIEW

File: 3 Reports.sql [hunk 1]
```diff
@@ -0,0 +1,26 @@
+--How many cars did we sell and how many cars did we lease?
+
+select count(*) as Amount, 
+    case c.isSold when 1 then 'Sold' else 'Leased' end as SoldOrLeased
+from Cars c
+where c.isSold is not null
+group by c.isSold
+
+--Which model earned the most revenue?
+select top 1 sum(c.PriceSold - c.PricePurchased) as Revenue, c.Model
```
Comment (yaakov1001) [inline 3 Reports.sql:10]: Revenue = `sum(c.PriceSold)`
Profit = `sum(c.PriceSold - c.PricePurchased)`

Comment (yaakov1001) [inline 3 Reports.sql]: Revenue = `sum(c.PriceSold)`
Profit = `sum(c.PriceSold - c.PricePurchased)`

**Review by yaakov1001 — COMMENTED** [review]
Excellent work. Please see comment in reports file - no need to fix.
Thank you very much.

File: 3 Reports.sql (hunk 1 above)
Comment (astrohlicpu) [reply 3 Reports.sql]: My mistake. Thanks for pointing it out.

**Review by yaakov1001 — COMMENTED** [review]
Everything is good now.
Grade: 100%

**Comment by astrohlicpu** [comment]
Great! You can merge and close the pull request.

**Pull request merged**